services/shhext/chat/encryption_test.go

```diff
@@ -29,7 +29,15 @@ type EncryptionServiceTestSuite struct {
 	bob   *EncryptionService
 }
 
-func (s *EncryptionServiceTestSuite) initDatabases() {
+func (s *EncryptionServiceTestSuite) initDatabases(baseConfig *EncryptionServiceConfig) {
+	os.Remove("/tmp/alice.db")
```

Comment:
I haven't noticed it earlier but `ioutil.TempDir` should be used: https://golang.org/pkg/io/ioutil/#TempDir. In this way, you can skip this call which introduced additional logic, does not handle errors etc.